app/util/mapLayerUtils.js

```diff
@@ -146,17 +146,7 @@ export const getMapLayerOptions = (options = {}) => {
     ...options,
   };
   lockedMapLayers.forEach(key => {
-    // Stop keyword locks every mode
-    if (key === 'stop') {
-      Object.keys(layerOptions[key]).forEach(subKey => {
-        if (layerOptions[key][subKey]) {
-          layerOptions[key][subKey].isLocked = true;
-          layerOptions[key][subKey].isSelected =
-            selectedMapLayers.includes(subKey) ||
-            selectedMapLayers.includes(key);
-        }
-      });
-    } else if (layerOptions.stop[key] && allModes.includes(key)) {
+    if (layerOptions.stop[key] && allModes.includes(key)) {
```

Comment:
I am not sure whether the `lockedMapLayers` are needed for StadtNavi at all or if the could removed from code  completely.

Reply:
We should not remove upstream functionality. Could you introduce a config variable like "enableLockedMapLayers" (with default = true if unset)